website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,46 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+Multi-selection enables users to select multiple rows in a table, facilitating bulk operations, actions, and editing. 
+
+Multi-selection consist of: 
+
+1. `Header Column / Select`: Used in the header row of the table. This acts as a master checkbox, enabling end-users to either select or deselect all rows on a single page simultaneously.
+
+![Example of multi-selection in a table header](/assets/components/table/multi-select-header.png)
+
+2. `Cell / Select`: Used in each table row. It allows for individual row selection and aligns with the Header Column component, reflecting the overall selection status of the table.
+
+![Example of multi-selection within table cells](/assets/components/table/multi-select-cells.png)
+
+!!! Info 
+
+In the Ember component, unlike the `cell / select` Figma component, achieving an indeterminate state is not possible. 
```

Comment:
I think what I'm trying to convey here is that the parent-child relationship is different in Figma vs. code. So if, in Figma, a consumer wanted to have an indeterminate state within one of the row columns, in code, they would not be able to do so. I think that maybe calling this out does not have much value as an `Info banner.`

Reply:
I can't imagine a situation in which a user could trigger an indeterminate state in a row or how that would have any value. I think unless/until we actually come up with a situation in which that would somehow be relevant or useful even mentioning it is confusing.

Reply:
Andrew, do you want to go ahead and delete this Info notice for now?

Reply:
Ah, I see, so in Figma, it _is_ possible to set a row's checkbox to indeterminate because it's reusing the `checkbox` component, even though that's not the intention or desired outcome.

I agree with @KristinLBradley that mentioning this may cause confusion, so let's omit it. If we find consumers use it incorrectly, we can help with education and follow up with clarification in the docs.

Reply:
Okay, yeah @KristinLBradley lets go ahead and delete this